app/pages/homepage/template.njk

```diff
@@ -38,25 +38,13 @@
         </li>
       {% endif %}
 
-      {% if (config.showCovid19 === 'true') %}
-        <li class="nhsuk-grid-column-one-half nhsuk-card-group__item" data-test-id="covid19-promo">
-          {{ card({
-            "href": "/solutions/covid19",
-            "headingLevel": 3,
-            "clickable": "true",
-            "heading": covid19PromoHeading,
-            "description": covid19PromoDescription
-          }) }}
-        </li>
-      {% endif %}
-
-      <li class="nhsuk-grid-column-one-half nhsuk-card-group__item" data-test-id="browse-promo">
+      <li class="nhsuk-grid-column-one-half nhsuk-card-group__item" data-test-id="vaccinations-promo">
         {{ card({
-          "href": "/solutions",
+          "href": "/solutions/vaccinations",
           "headingLevel": 3,
           "clickable": "true",
-          "heading": viewSolutionsPromoHeading,
-          "description": viewSolutionsPromoDescription
+          "heading": vaccinationsPromoHeading,
+          "description": vaccinationsPromoDescription
         }) }}
       </li>
```

Comment:
Do we know for certain that the vaccinations tile shouldn't be toggleable using the `showCovid19` feature flag?

I've added a question to the story discussion thread for this.